### PR TITLE
fix: add read me file, manual test and fix type

### PR DESCRIPTION
**Pull request by mr25mr**
This PR address
* readme file
* Adapt property binding info manual test
* small correction for ArrayType which can be `UI5Typedef`

### CODE REVIEW

**Review by vinayhospete — APPROVED** [review]

File: packages/vscode-ui5-language-assistant/README.md [hunk 1]
```diff
@@ -134,6 +134,17 @@ The feature is available in the following:
 
   - Enum values
   - SAPUI5 Namespaces
+  - any keys of property binding info
```
Comment (MN-public) [inline packages/vscode-ui5-language-assistant/README.md:137]: Do we need "any" here? Please also start with the capital letter to follow the pattern above

File: packages/vscode-ui5-language-assistant/README.md [hunk 2]
```diff
@@ -134,6 +134,17 @@ The feature is available in the following:
...
+
+### Semantic highlighting for property binding info or aggregation binding
+
+![](https://raw.githubusercontent.com/SAP/ui5-language-assistant/master/packages/vscode-ui5-language-assistant/resources/readme/preview-semantic-highlighting-binding.png)
+
+#### Description
+
+Make sure that semantic highlighting is not disabled. For details check [here](https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#enablement-of-semantic-highlighting)
+
+Property binding info or aggregation binding is semantically highlighted.
```
Comment (MN-public) [inline packages/vscode-ui5-language-assistant/README.md:147]: Property binding info and aggregation binding are displayed semantically highlighted to let you easily grasp the sense of dense code.
Note: Make sure that semantic highlighting is enabled. For more information, see [[Enablement of semantic highlighting](https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#enablement-of-semantic-highlighting)

File: packages/vscode-ui5-language-assistant/README.md [hunk 3]
```diff
@@ -134,6 +134,17 @@ The feature is available in the following:
 
   - Enum values
   - SAPUI5 Namespaces
+  - any keys of property binding info
+
+### Semantic highlighting for property binding info or aggregation binding
```
Comment (MN-public) [inline packages/vscode-ui5-language-assistant/README.md:139]: ### Semantic Highlighting

**Review by MN-public — CHANGES_REQUESTED** [review]

File: packages/vscode-ui5-language-assistant/README.md [hunk 4]
```diff
@@ -134,6 +134,17 @@ The feature is available in the following:
 
   - Enum values
   - SAPUI5 Namespaces
+  - Name of property binding info
+
+### Semantic Highlighting for property binding info or aggregation binding
```
Comment (MN-public) [inline packages/vscode-ui5-language-assistant/README.md:139]: I would remove everything after Semantic Highlighting here

**Review by MN-public — CHANGES_REQUESTED** [review]

**Review by MN-public — APPROVED** [review]

**Pull request merged**